### ISSUE TITLE
REQ: Set background color/transparency of thumbnail on ComponentDefinition#save_as

### ISSUE THREAD

**Issue by esterkimx** [SketchUp/api-issue-tracker#441]
When we save components in SketchUp, it always renders thumbnails with white background. It would be really helpful to be able to customize it (especially making it transparent), so it would look nice on any background in HtmlDialogs.

Currently a thumbnail is just a PNG image embedded inside SKP file, so I guess it would be really easy to add an alpha channel to it.

#### Possible implementations

Add a keyword argument to ComponentDefinition#save_as:
```ruby
definition.save_as('component.skp', bg_color: Sketchup::Color.new(0, 0, 0, 0))
```

Or even just be able to make a background 100% transparent
```ruby
definition.save_as('component.skp', transparent: true)
```

**Comment by Eneroth3** [comment]
I think the easiest solution would just be transparent and then whatever interface is using the image can set the background of choice.

**Comment by Matherone** [comment]
Yes, Transparent, or Alpha 0.0, is much **better default background** for thumbnails than white. 

The host application may then set the the background to whatever color its local UI theme calls for. For instance light grey, dark grey, black, or anything.

Many end users rely on visuals to distinguish components from each other.

Our clients are sending us custom asset libraries comprising hundreds, or thousands, of components. 

Transparent thumbnail backgrounds make it easier for our end users to **recognize** and **find** the correct components when browsing collections small and large.

Compared to glaring white, non-white backgrounds can also make the subject matter **stand out** better.

Would you please implement transparent thumbnail backgrounds?
